src/v/cluster/scheduling/leader_balancer.cc

```diff
@@ -440,8 +398,8 @@ leader_balancer::find_leader_shard(const model::ntp& ntp) {
         return std::nullopt;
     }
 
-    // If the inital query was for a non_replicable ntp, the get_leader() call
-    // would have returned its source topic
+    // If the inital query was for a non_replicable ntp, the get_leader()
+    // call would have returned its source topic
```

Comment:
These re-wraps of the last words on lines are usually a sign that at some point the code was accidentally indented while you edited it, then clang-format'd on safe -- please clean up so that the patch is clearer.

Reply:
Cleaned